app/serializers.py

```diff
@@ -613,6 +613,11 @@ def create(self, validated_data):
             emp_ser.save(user=user, organization=organization, user_allowed=True)
         return user
 
+    def validate_email(self, value):
+        if models.User.all_objects.filter(email=value).exists():
```

Comment:
Can you also. ensure that `email` field has a unique index?

Reply:
sure